keyboards/sck/m0116b/keymaps/m0118/keymap.c

```diff
@@ -0,0 +1,118 @@
+/* Copyright 2018 jrfhoutx
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+ /* Keymap BASE: (Base Layer) Default Layer
+   * ,---------------------------------------------------------------------------------.
+   * |                     |  SAFE_RANGE  |                                            |
```

Comment:
```suggestion
   * |                     |  M0116B  |                                                |
```

Reply:
None of the keymaps  compile when using `M0116B` instead of `SAFE_RANGE`. they all compile perfectly fine, and the sendstring works correctly, when using `SAFE_RANGE`, which is why that is what is in in those spaces

Reply:
corrected